tests/unit/Serializers/ReferenceListSerializerTest.php

```diff
@@ -65,10 +65,15 @@ public function serializationProvider() {
 					array(
 						'hash' => 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 						'snaks' => array()
+					),
+					array(
+						'hash' => 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
+						'snaks' => array()
```

Comment:
Same as above. How is it possible that a Reference with 1 snak is serialized as an empty array?